extension/js/common/composer.ts

```diff
@@ -1707,6 +1718,20 @@ export class Composer {
     }
   }
 
+  private async checkEmailAliases() {
+    if (!this.urlParams.isReplyBox) {
+      const addresses = Value.arr.unique((await Settings.fetchAcctAliasesFromGmail(this.urlParams.acctEmail)).concat(this.urlParams.acctEmail));
+      const storedAdresses = (await Store.getAcct(this.urlParams.acctEmail, ['addresses'])).addresses!;
+      if (addresses.sort().join() !== storedAdresses.sort().join()) { // This way of comparation two arrays works only for not object arrays
+        await Store.setAcct(this.urlParams.acctEmail, { addresses });
+        if (await Ui.modal.confirm('Your email aliases on Gmail have refreshed since the last time you used FlowCrypt.\nReload the compose window now?')) {
+          this.app.closeMsg();
+          BrowserMsg.send.openNewMessage(this.urlParams.parentTabId);
```

Comment:
`window.location.reload()` will work better instead of these two lines.

Reply:
Oh, Yes, it is definitely better. 
I forgot that after `location.reload` the whole window won't reload, it reloads the only iframe
Thanks!